src/main/resources/application.yml

```diff
@@ -1,3 +1,11 @@
 spring:
   application:
     name: wcc-platform
+
+springdoc:
```

Comment:
is there a command line to update or something like this? Could you include in README maybe?

Reply:
it is a set-up for swagger.. once it is in the code it will be available to use. Yes I will add to the README how to use the swagger endpoint with this repo.

Reply:
> is there a command line to update or something like this? Could you include in README maybe?

it is a set-up for swagger.. once it is in the code it will be available to use. Yes I will add to the README how to use the swagger endpoint with this repo.